public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
```

Comment:
we put this note in a lot of places, do we actually need this or it is quite obvious for users? also, non Git DVC repo probably is an advanced case and its users will have even better understanding for this mode limitations?

Reply:
The note is only here and in the `dvc diff` cmd ref. But yes, it's obvious now because I'm using the term DVC repository more. Removing in both places!